packages/transaction-controller/src/EtherscanRemoteTransactionSource.ts

```diff
@@ -143,10 +124,10 @@ export class EtherscanRemoteTransactionSource
       chainId: currentChainId,
       hash: txMeta.hash,
       id: random({ msecs: time }),
-      networkID: currentNetworkId,
       status: TransactionStatus.confirmed,
       time,
       transaction: {
+        chainId: currentChainId,
```

Comment:
Do we want chainId twice here? Both on the root object and inside the transaction object?

Reply:
that's just how it's been done. The `transaction` object is essentially the params used to create the unsigned transaction. We could normalize it into a single location, but that's definitely something to be dealt with in another PR. We'd also want to contact the Confirmations team since this is their domain (and I don't want to step on their feet again since I already did making these changes without notifying them first)

Reply:
It's going to be renamed to `txParams` in this PR: https://github.com/MetaMask/core/pull/1651/files

Reply:
oof looks like that PR got merged and now this requires a rebase 😭